etc/profile-m-z/torbrowser-launcher.profile

```diff
@@ -33,9 +33,9 @@ whitelist ${HOME}/.local/share/torbrowser
 whitelist /opt/tor-browser
 whitelist /usr/share/torbrowser-launcher
```

Comment:
Maybe we can include disable process.

Reply:
Doesn't seem to cause any issues. Let's add it!